canister/src/api/get_utxos.rs

```diff
@@ -913,50 +913,49 @@ mod test {
 
     #[test]
     fn get_utxos_does_not_include_other_addresses() {
-        for network in [Network::Mainnet, Network::Testnet, Network::Regtest].iter() {
```

Comment:
We can no longer create fake testnet and mainnet blocks as these fail the validation, so this test (and others) are now only generating regtest blocks.